### PR TITLE
fix(tests): check that content in all expected languages is available

### PR DESCRIPTION
**Pull request by jrgm**
This is a test that I wrote I don't know how long ago, but I've been running this against stage with each release. (I think I wrote this after some release where this test would have detected a l10n problem).

Anyways, seems useful to have this as part of regular CI tests.

### CODE REVIEW

File: tests/server/lang.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      headers: {
+        'Accept-Language': lang
+      }
+    };
+
+    suite['#https get ' + httpsUrl + '/ -> ' + lang] = function () {
+      var dfd = this.async(intern.config.asyncTimeout);
+      request(httpsUrl + '/', options, dfd.callback(function (err, res) {
+        assert.ok(!err);
+        assert.equal(res.statusCode, 200);
```
Comment (zaach) [inline tests/server/lang.js]: Should we do a check here that the language attribute shows up correctly on the HTML tag? E.g. `      assert.ok(res.body.match(/lang="zh-CN"/));`.

Comment (jrgm) [inline tests/server/lang.js]: Good idea. I added that check.

**Comment by zaach** [comment]
:+1:

**Pull request merged**